pkg/pager/model.go

```
@@ -0,0 +1,381 @@
+}
+
+var (
+	pagerHelpHeight int
+
+	// Logo
+	logo = statusBarHelpStyle(" \U0001F47D  ")
+
+	mintGreen = lipgloss.AdaptiveColor{Light: "#89F0CB", Dark: "#89F0CB"}
+	darkGreen = lipgloss.AdaptiveColor{Light: "#1C8760", Dark: "#1C8760"}
+
+	lineNumberFg = lipgloss.AdaptiveColor{Light: "#656565", Dark: "#7D7D7D"}
```

golangci-lint (1.64.8)

[error] var `lineNumberFg` is unused

(unused)